openmdao/components/exec_comp.py

```diff
@@ -131,7 +140,7 @@ def __init__(self, exprs, **kwargs):
         -----
         If a variable has an initial value that is anything other than 0.0,
```

Comment:
Should this be reworded with the initial value change? "If a variable has an initial value that is anything other than  0.0 ..."

Reply:
fixed